modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -230,6 +230,14 @@ type: kubernetes.io/tls
 
 Applying this label to your secret enables your ingress controller to immediately apply the updated TLS material into application namespaces. If you don't add this label, you must manually update the secret in each namespace.
 
+== Modifying an Existing Ingress Template
```

Comment:
According to our MuleSoft CX Writing Style Reference (https://confluence.internal.salesforce.com/x/l5POEw), we should use gerunds in H1 and infinitives in H2+.